test/quantization/fx/test_quantize_pt2e.py

```diff
@@ -190,6 +192,74 @@ def forward(self, x):
             code_after_recompile = m.code
             self.assertTrue(code_before_recompile == code_after_recompile, error_msg)
 
+    @skipIfNoX86
+    @xfailIfPython311
+    def test_inductor_backend_config_conv(self):
```

Comment:
can you create a new test case for this test? e.g. TestQuantizePT2EOneDNN

Reply:
Done.